tests/test_e2e.py

```diff
@@ -322,6 +322,12 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.session.close()
 
+    @classmethod
+    def get_lab_app_object(cls, **query ):
```

Comment:
Does this API also have an online document? If so, let's leave its URL as comment, pretty much [like what we did here](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/165/files#diff-2f43f6151484c4ac714c0be06ac04a7cR342). That would be the only way to let the future maintainer know what those `**query` would be.